internal/export/distro/client_test.go

```diff
@@ -27,6 +28,12 @@ const (
 	invalidRegistrationList2 = "[" + registrationInvalidStr + "," + registrationStr + "]"
 )
 
+func init() {
+	if LoggingClient == nil {
+		LoggingClient = logger.NewClient(internal.ExportDistroServiceKey, false, "")
+	}
+}
+
```

Comment:
My preference would be to setup these kinds of dependencies in a TestMain function. See internal/core/data/event_test.go